KubeArmor/enforcer/appArmorProfile.go

```diff
@@ -24,13 +24,17 @@ func (ae *AppArmorEnforcer) ResolvedProcessWhiteListConflicts(processWhiteList *
 	numOfRemovedElements := 0
 
 	for index, line := range *processWhiteList {
+		processed := false
 		for source := range fromSources {
 			if strings.Contains(line, source) {
 				*fusionProcessWhiteList = append(*fusionProcessWhiteList, source)
 
-				// remove line from WhiteList
-				prunedProcessWhiteList = kl.RemoveStringElement(prunedProcessWhiteList, index-numOfRemovedElements)
-				numOfRemovedElements = numOfRemovedElements + 1
+				if !processed {
+					// remove line from WhiteList
+					prunedProcessWhiteList = kl.RemoveStringElement(prunedProcessWhiteList, index-numOfRemovedElements)
```

Comment:
@seswarrajan, I am still not clear as to what was causing the segfault here?
This PR adds a check to call `kl.RemoveStringElement()` just once ... but even if it is called multiple times, how would it result in a segfault?